depth_filter:
  ros__parameters:
    vis: false

    fov_filter:
      enabled: false
      top_angle: 15.0
      bottom_angle: 15.0

    median_filter:
      enabled: false
      kernel_size: 5

    dist_filter:
      enabled: false
      min_dist: 1.0
      max_dist: 10.0